rust/core/src/driver_manager.rs

```diff
@@ -184,6 +200,65 @@ impl ManagedDriver {
         Ok(ManagedDriver { inner })
     }
 
+    /// Load a driver either by name, filename, path, or via locating a toml manifest file.
+    /// The LoadFlags control what directories are searched to locate a manifest.
+    pub fn find_load_from_name(
+        name: impl AsRef<OsStr>,
+        entrypoint: Option<&[u8]>,
+        version: AdbcVersion,
+        load_flags: LoadFlags,
+    ) -> Result<Self> {
+        let driver_path = Path::new(name.as_ref());
+        let allow_relative = load_flags & LOAD_FLAG_ALLOW_RELATIVE_PATHS != 0;
+        let ext = driver_path.extension();
+        if ext.is_some() {
```

Comment:
```suggestion
        if let Some(ext) = driver_path.extension() {
```

There's generally little reason to use is_some.